src/main/core/Messenger/ExecuteScheduledTask.php

```diff
@@ -9,43 +9,52 @@
  * file that was distributed with this source code.
  */
 
-namespace Claroline\MessageBundle\Listener;
+namespace Claroline\CoreBundle\Messenger;
 
+use Claroline\AppBundle\Persistence\ObjectManager;
 use Claroline\CoreBundle\Entity\Task\ScheduledTask;
-use Claroline\CoreBundle\Event\GenericDataEvent;
+use Claroline\CoreBundle\Manager\MailManager;
 use Claroline\CoreBundle\Manager\Task\ScheduledTaskManager;
 use Claroline\MessageBundle\Manager\MessageManager;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
 
-class MessageListener
+class ExecuteScheduledTask implements MessageHandlerInterface
```

Comment:
Should be `ExecuteScheduledTaskHandler`.

Reply:
done, thx :)